dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -540,6 +541,16 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
                 .collect(Collectors.toCollection(LinkedHashSet::new));
     }
 
+    @GET
+    @Timed
+    @UnitOfWork
+    @Path("/starredOrganizations")
+    @ApiOperation(value = "Get the logged-in user's starred organizations.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Organization.class, responseContainer = "List")
+    public Set<Organization> getStarredOrganizations(@ApiParam(hidden = true) @Auth User user) {
+        User u = userDAO.findById(user.getId());
```

Comment:
Is this line necessary? It might be due to some Hibernate thing, but it seems unnecessary to me.

Reply:
The DAO is probably necessary or at least follows the convention used by all other user calls in this class, but yeah it can probably just be simplified to return in the same line.